relayer/crates/starknet-chain-components/src/components/starknet_to_cosmos.rs

```diff
@@ -54,6 +53,9 @@ mod preset {
                 CreateClientPayloadBuilderComponent,
                 UpdateClientPayloadBuilderComponent,
                 ChannelOpenInitMessageBuilderComponent,
+                ChannelOpenTryMessageBuilderComponent,
```

Comment:
I think we can do this for Connection handshake too. Note, this closes #268.

Reply:
I tried but it didn't work. Seems like there are some other conflicting constraints. We can fix that in separate PR.